src/java.desktop/share/classes/javax/swing/JFormattedTextField.java

```diff
@@ -1086,23 +1086,27 @@ protected void setEditValid(boolean valid) {
         }
 
         /**
-         * Subclass and override if you wish to provide a
+         * Subclass must override if you wish to provide a
```

Comment:
It think the original text was correct: Subclass was used as a verb in imperative. You have to *subclass* before you can override.